### PR TITLE
Add ability to define custom server url

### LINKED ISSUE #1: Handle deploying with a custom server URL
With [this PR](https://github.com/wasp-lang/wasp/pull/2234) merged, we will be able to define a custom server URL for the react app (client). Earlier, this deploy action would default to using the fly.dev domain.

### PR DESCRIPTION
**Pull request by Case-E**
fixes #1 

Can only be merged after https://github.com/wasp-lang/wasp/pull/2234

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Add your Fly.io token as a repository secret e.g. calling it `FLY_TOKEN`
 
 Read more about how to do it: https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository
 
+### Define a custom server URL
+If you've add a custom domain to your server you can define it as a repository secret `SERVER_URL`. When you don't define a server URL, wasp might default/override to using the `<app-name>-server.fly.dev` address for the server.
```
Comment (Case-E) [inline README.md:23]:
```suggestion
If you've added a custom domain to your server you can define it as a repository secret `SERVER_URL`. When you don't define a server URL, wasp might default/override to using the `<app-name>-server.fly.dev` address for the server. 
```

**Comment by hellwaiver** [comment]
Is this working?

**Comment by Case-E** [comment]
@hellwaiver You should be able to use the workaround I have here: https://github.com/wasp-lang/wasp/pull/2234

Those changes are merged but pending a release. Once the changes are released on Wasp, this MR can be merged.

**Comment by Case-E** [comment]
Tagging @Martinsos @infomiho to see if they have any insights here.

**Comment by infomiho** [comment]
@Case-E This looks good to me 👍 let's merge it after the 0.15.0 release, as you said :)

**Comment by Case-E** [comment]
@infomiho Now that 0.15.0 is released, can we test and merge this?

**Comment by infomiho** [comment]
@Case-E sorry for dropping the ball on this! I think this is ready to be merged.

**Pull request merged**